terraform/staging-account/tfstate.json

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
Is the state file supposed to be included?

Reply:
Removed 👍